src/canvas-extractor/tests/graphql/assignments/test_extractor_assignments.py

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+
+from pandas import DataFrame
+from edfi_canvas_extractor.graphql.assignments import assignments_synced_as_df
+
+
+def test_assignments(gql_no_run, api):
+    """
+    Get from the sample data
+    obtain the assignments info
+    Check and check the return type
+    """
+    gql_no_run.extract(api)
+    assignments = gql_no_run.assignments
```

Comment:
Here and in similar places in other tests for `assignments, courses, enrollments, sections and students` , we directly use `gql_no_run.assignments`, shouldn't we use the `get_assignments` functions in the  `GraphQLExtractor` class.